core/src/main/res/values/styles.xml

```diff
@@ -235,6 +235,12 @@
         <item name="android:fontFamily">sans-serif-light</item>
     </style>
 
+    <style name="AntennaPod.TextView.HeadingSemiBold" parent="@android:style/TextAppearance.Medium">
```

Comment:
Is there already a material style that does something similar? Then we should use that one instead

Reply:
Nope, the font size is changed!

Reply:
This is a new style there is no such , please mark this as accepted

Reply:
Then I guess we should use the font size that Google recommends for Material3?

Reply:
this is a middle value of both so not an issue, we can play with the font size as there are multiple levels of it